fork-off/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # client
 [dependencies.substrate-api-client]
 git = "https://github.com/Cardinal-Cryptography/substrate-api-client.git"
-tag = "fixed_dependencies"
+tag = "polkadot-v0.9.13.b2bfc0c3"
```

Comment:
best confirm if these changes are backwards compat with encoding on current testnet (can the tool still communicate with it?), or confirm with @bartoszjedrzejewski this does not hinder with their work on devnet

Reply:
I'll confirm with @bartoszjedrzejewski

Reply:
Actually, I confirmed running locally, and it works.